cmds/deploy.go

```diff
@@ -217,6 +223,7 @@ func NewCmdDeploy(f *cmdutil.Factory) *cobra.Command {
 		},
 	}
 	cmd.PersistentFlags().StringP("domain", "d", defaultDomain(), "The domain name to append to the service name to access web applications")
+	cmd.PersistentFlags().StringP("apidomain", "", "--domain", "if set the fabric8 pod uses that domain to call the console")
```

Comment:
This should be `cmd.PersistentFlags().String("api-server", defaultDomain(), "API server address")` or something like that.